QandA/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Meetings(models.Model):
+    meeting_id = models.AutoField(primary_key=True)
+    HOMER = 'H'
+    PLATO = 'P'
+    JARVIS = 'J'
+    FRIDAY = 'F'
+    VENUE_CHOICES = (
+        (HOMER,'homer'),
+        (PLATO,'plato'),
+        (JARVIS,'jarvis'),
+        (FRIDAY,'friday'),
+    )
+    venue = models.CharField(max_length=50,choices=VENUE_CHOICES,default=JARVIS)
+    moderator_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    start_time = models.DateTimeField(help_text="Please use the following format: <em>YYYY-MM-DD</em>.")
+    end_time = models.DateTimeField(help_text="Please use the following format: <em>YYYY-MM-DD</em>.")
+
+
+class Question(models.Model):
+    question_id = models.AutoField(primary_key=True)
+    question_text = models.CharField(max_length=100)
+    meeting_id = models.ForeignKey(Meetings,on_delete=models.CASCADE)
+    user_id = models.ForeignKey(User,on_delete=models.CASCADE)
+    YES = 'Y'
+    NO = 'N'
+    ANON_CHOICES = (
+        (YES,'yes'),
+        (NO,'no'),
+    )
+    anon_status = models.CharField(max_length=3,choices=ANON_CHOICES,default=NO)
```

Comment:
Same comment as Char vs SmallPositiveInteger as pointed before.

+ Now if we have field  as CharField we should have max_length as 1 vs 3 (now,) because we're only storing Y/N here.

Reply:
How about a BooleanField for this case?